teuthology/suite/util.py

```diff
@@ -271,6 +274,13 @@ def package_version_for_hash(hash, kernel_flavor='basic', distro='rhel',
             sha1=hash,
         ),
     )
+
+    if bp.distro == CONTAINER_DISTRO and bp.flavor == CONTAINER_FLAVOR:
+        log.info('container build %s, checking for build_complete' % bp.distro)
+        if not bp.build_complete:
+            log.info('build not complete')
+            return None
```

Comment:
Because this behavior isn't configurable, I'm a little concerned it could break others' workflow. Or can we be sure everyone will want to block/fail on container builds?

Reply:
I debated it.  The worst it should do, in the "successful build" case, is fail and delay the testing for another 10 minutes (or choose the second-newest success with --newest).  I don't have strong opinions though, and it could be Yet Another Commandline Switch I suppose.